test/integration/host_test.rb

```diff
@@ -51,45 +42,6 @@ def class_params
       assert page.has_link?("Additional Information", :href => "#info")
     end
 
-    test "default primary interface is in the overview table" do
-      assert_new_button(hosts_path, "New Host", new_host_path)
-
-      # switch to interfaces tab
-      page.find(:link, "Interfaces").click
-
-      # test column content
-      assert table.find('td.identifier', :visible => true).has_content?('')
-      assert table.find('td.type', :visible => true).has_content?('Interface')
-      assert table.find('td.mac', :visible => true).has_content?('')
-      assert table.find('td.ip', :visible => true).has_content?('')
-      assert table.find('td.fqdn', :visible => true).has_content?('')
-
-      # should have table header and the primar interface row
-      assert_equal 2, table.all('tr', :visible => true).count
-
-      # test the tlags are set properly
-      assert table.find('td.flags .primary-flag.active')
-      assert table.find('td.flags .provision-flag.active')
-    end
-
-    test 'choosing a hostgroup overrides other host attributes' do
-      original_hostgroup = FactoryGirl.
-        create(:hostgroup, :environment => FactoryGirl.create(:environment))
-      overridden_hostgroup = FactoryGirl.
-        create(:hostgroup, :environment => FactoryGirl.create(:environment))
-
-      visit new_host_path
-      select2(original_hostgroup.name, :from => 'host_hostgroup_id')
-      wait_for_ajax
-
-      click_on_inherit('environment')
-      select2(overridden_hostgroup.name, :from => 'host_hostgroup_id')
-      wait_for_ajax
-
-      environment = find("#s2id_host_environment_id .select2-chosen").text
-      assert_equal overridden_hostgroup.environment.name, environment
-    end
-
     describe "NIC modal window" do
       setup { skip "Temporarily disabled until issue #9138 gets resolved" }
```

Comment:
These NIC tests should also probably move since they rely on the JS driver.  Knowing what we know now, we could probably fix them too.